zsh-aws-vault.plugin.zsh

```diff
@@ -13,6 +13,35 @@ function avsh() {
   aws-vault exec $1 -- zsh
 }
 
+function aws-login() {
```

Comment:
I've got this defined above as `avl`, I like how short and concise it is compared to `aws-login`. I do worry a bit about conflicts, but I didn't have any. I think I prefer `avl` to this.